sunpy/net/dataretriever/sources/aia_synoptic.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+import re
+from datetime import datetime, timedelta
+from sunpy.net.dataretriever import GenericClient
+from sunpy.net import attrs as a
+import astropy.units as u
+from sunpy.net.dataretriever.client import QueryResponse
+
+__all__ = ["AIASynopticClient"]
+
+class AIASynopticClient(GenericClient):
+    """
+    A client for retrieving AIA synoptic data from JSOC.
+
+    This client retrieves synoptic AIA data from the following source:
+    https://jsoc1.stanford.edu/data/aia/synoptic/
+
+    The synoptic dataset includes lower-resolution (1k) images, with additional
+    image processing steps like downsampling and time integration.
+    """
+
+    baseurl = "https://jsoc1.stanford.edu/data/aia/synoptic/"
+    known_wavelengths = [94, 131, 171, 193, 211, 304, 335, 1600, 1700, 4500]
+
+    required = {a.Time, a.Instrument}
+    optional = {a.Sample, a.Level, a.Wavelength, a.ExtentType}
+
+    @property
+    def info_url(self):
+        return self.baseurl
+
+    @classmethod
+    def register_values(cls):
+        adict = {
+            a.Instrument: [("AIA", "Data from the Atmospheric Imaging Assembly instrument.")],
+            a.Physobs: [("intensity", "Brightness or intensity of the solar atmosphere at different wavelengths.")],
+            a.Source: [("SDO", "The Solar Dynamics Observatory.")],
+            a.Wavelength: [(f"{wv:04d}", f"{wv} Å") for wv in cls.known_wavelengths],
+            a.Provider: [("JSOC", "Joint Science Operations Center at Stanford.")],
+            a.Level: [("synoptic", "Level 1.5 data processed for quicker analysis.")],
```

Comment:
```suggestion
            a.Level: [("1.5", "Level 1.5 data processed for quicker analysis.")],
```

Reply:
I guess I was hoping to maybe just use the Level attribute. It seems appropriate: it labels a homogenious data set from JSOC with standardized processing steps and observational parameters. By referencing the synoptic level of AIA data you are specifying this in a way users probably already know to do. But this is just my preferred implementation and if you'd like it more standardized I understand.

Reply:
Hmm, fair. I will have to let the others chime in on this matter before we make a change here.